drivers/InterruptIn.h

```diff
@@ -66,6 +66,12 @@ class InterruptIn : private NonCopyable<InterruptIn> {
      *  @param pin InterruptIn pin to connect to
      */
     InterruptIn(PinName pin);
+    /** Create an InterruptIn connected to the specified pin,
```

Comment:
You may want a line break here; same between the constructor and the destructor.

Reply:
I tried to match the existing style. Doesn't matter to me which way you prefer it. Let me know, or feel free to change it yourself.